contributors/design-proposals/cloudprovider-storage-metrics.md

```diff
@@ -0,0 +1,48 @@
+# Cloud Provider (specifically GCE and AWS) metrics for Storage API calls
+
+## Goal
+
+Kubernetes should provide metrics such as - count & latency percentiles
+for cloud provider API it uses to provision persistent volumes.
+
+In a ideal world - we would want these metrics for all cloud providers
+and for all API calls kubernetes makes but to limit the scope of this feature
+we will implement metrics for:
+
+* GCE
+* AWS
+
+We will also implement metrics only for storage API calls for now. This feature
+does introduces hooks into kubernetes code which can be used to add additonal metrics
+but we only focus on storage API calls here.
+
+## Motivation
+
+* Cluster admins should be able to monitor Cloud API usage of Kubernetes. It will help
+  them detect problems in certain scenarios which can blow up the API quota of Cloud
+  provider.
+* Cluster admins should also be able to monitor health and latency of Cloud API on
+  which kubernetes depends on.
+
+## Implementation
+
```

Comment:
Can you introduce what the metric collection framework is (Prometheus) and description or link to how it is integrated into Kubernetes (does it work for all deployments?), etc.